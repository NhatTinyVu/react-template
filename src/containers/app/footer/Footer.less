.footer {
  text-align: center;
  align-self: flex-end;
  width: 100%;
  flex: initial;
}