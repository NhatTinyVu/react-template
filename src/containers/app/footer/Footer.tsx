import React from 'react';
import { Layout, Icon } from 'antd';
import styles from './Footer.less';

const Footer: React.FC = () => (
  <Layout.Footer className={styles.footer}>
    <a href="https://github.com/NhatTinyVu/react-template">
      <Icon type="github" />
      &nbsp;Github
    </a>
  </Layout.Footer>
);

export default Footer;
