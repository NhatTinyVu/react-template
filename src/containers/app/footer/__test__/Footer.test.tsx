import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '..';

describe('Footer', () => {
  test('renders correctly', (): void => {
    const wrapper = shallow(<Footer />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
