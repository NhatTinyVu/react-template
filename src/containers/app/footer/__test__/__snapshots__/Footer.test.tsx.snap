// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<Adapter
  className="footer"
>
  <a
    href="https://github.com/NhatTinyVu/react-template"
  >
    <Icon
      type="github"
    />
    Â Github
  </a>
</Adapter>
`;
